kernel/init.c

```diff
@@ -67,10 +67,10 @@ u64_t __noinit __idle_time_stamp;  /* timestamp when CPU goes idle */
 #endif
 
 #ifdef CONFIG_EXECUTION_BENCHMARKING
-u64_t __noinit __start_swap_tsc;
-u64_t __noinit __end_swap_tsc;
-u64_t __noinit __start_intr_tsc;
-u64_t __noinit __end_intr_tsc;
+u64_t __noinit __start_swap_time;
+u64_t __noinit __end_swap_time;
+u64_t __noinit __start_intr_time;
+u64_t __noinit __end_intr_time;
 u64_t __noinit __start_tick_tsc;
```

Comment:
what about these 2?

Reply:
agreed, @youvedeep not sure what tsc stood for, but it seems like you forgot to rename a couple

Reply:
hi @carlescufi / @galak 

Yes i missed this while changing the name of other variables used. I will include this in my next version of patch.  Also as 32KHz is too slow to get the benchmark data, So we need to use timer0 for this with prescale value of 0(16MHz).

I will put new patch for review.